backend/src/services/exam/model.js

```diff
@@ -1,41 +1,51 @@
 const mongoose = require('mongoose');
 
 const ExamSchema = new mongoose.Schema({
-  title: {
-    type: String,
-    required: true,
-    trim: true,
-  },
-  instruction: {
-    type: String,
-    required: false,
-    trim: true,
-  },
-  reminder: {
-    type: String,
-    required: false,
-    trim: true,
-  },
-  exercices: [{
-    type: mongoose.Schema.Types.ObjectId,
-    ref: 'Exercice',
-  }],
-  creationTime: {
-    type: Date,
-    default: () => Date.now(),
-  },
-  author: {
-    type: mongoose.Schema.Types.ObjectId,
-    trim: true,
-    // ref: 'author', CF service Teamy
-  },
-  session: {
-    type: mongoose.Schema.Types.ObjectId,
-    // ref: 'Session', CF service Teamy
-    // gives us the dates,
-    // gives us the subject
-    trim: true,
-  },
+    title: {
+        type: String,
+        required: true,
+        trim: true,
+    },
+    instruction: {
+        type: String,
+        required: false,
+        trim: true,
+    },
+    reminder: {
+        type: String,
+        required: false,
+        trim: true,
+    },
+    showScale: {
+        type:Boolean,
```

Comment:
A whitespace between 'type:' and 'Boolean' would look better

Reply:
Also, if showscale has a default value, we can remove the 'required: false', don't you think ?

Reply:
@ValentinChion